### PR TITLE
modified log_level in terraform

### PR DESCRIPTION
**Pull request by sijegbulem-ch**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No IaC security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: terraform/groups/ecs-service/locals.tf
Comment (gjackson-ch) [inline terraform/groups/ecs-service/locals.tf:1]: Extra comma may cause issues

**Review by gjackson-ch — CHANGES_REQUESTED** [review]

File: terraform/groups/ecs-service/locals.tf
Comment (sijegbulem-ch) [reply terraform/groups/ecs-service/locals.tf:1]: sorted

**Comment by ch-code-analysis** [comment]
CI: No IaC security warnings found

**Review by gjackson-ch — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**